packages/web/src/features/users/InviteUserView.tsx

```diff
@@ -23,10 +24,13 @@ export default function InviteUserView() {
   useTitle(t('common:tab_titles.invite_user'));
   const flash = useFlash();
 
+  const queryClient = useQueryClient();
+
   const formContext = useForm<FormData>();
   async function onSubmit({ email }: FormData) {
     try {
       await apiClient.users.invite({ email });
+      queryClient.invalidateQueries(['session']);
```

Comment:
We shouldn't need to refetch the session when you invite a new user because that shouldn't ever affect the current user

Reply:
Whoops, I should've removed that. Done.